packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -247,10 +246,10 @@ test.describe('PieCookieBanner - Component tests', () => {
         await page.click(managePreferencesSelector);
         await page.click(getPreferenceItemSelector('all')); // turn on all nodes
         await page.click(getPreferenceItemSelector('functional')); // turn off one of the preferences
-        const isToggleAllChecked = await page.locator(getPreferenceItemSelector('all')).isChecked();
+        const isToggleAllChecked = await page.locator(getPreferenceItemSelector('all'));
```

Comment:
same issue here regarding the variable name sounding like it's a boolean

Reply:
Fixed.